Frameworks/CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -1118,6 +1128,11 @@ void __CFInitialize(void) {
         memset(__CFRuntimeClassTable, 0, sizeof(__CFRuntimeClassTable));
         memset(__CFRuntimeObjCClassTable, 0, sizeof(__CFRuntimeObjCClassTable));
 
+#if WINOBJC
+        for (CFIndex idx = 1; idx < __CFRuntimeClassTableSize; ++idx) {
+            __CFRuntimeObjCClassTable[idx] = (uintptr_t)&_OBJC_CLASS__NSCFType;
+        }
```

Comment:
I'd rather not get my peanut butter in my chocolate over here :)

---
In reply to: [81260899](https://github.com/Microsoft/WinObjC/pull/1059#discussion_r81260899) [](ancestors = 81260899)